Containers/nextcloud/Dockerfile

```diff
@@ -91,11 +91,12 @@ RUN set -ex; \
 # set recommended PHP.ini settings
 # see https://docs.nextcloud.com/server/stable/admin_manual/configuration_server/server_tuning.html#enable-php-opcache
```

Comment:
The documentation is wrong at this point. Not sure about early PHP 8.0 versions, but all 8.1/8.2/8.3 versions do not use more than a little above 1 MiB of this buffer (with one NC instance), and since OPcache size overall is 128 MiB, using 128 MiB for JIT alone cannot work.

It should be also noted that JIT in general works on x86 only, not on ARM.

Reply:
could the documentation then be updated please?